buildAndReleaseTask/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 0,
         "Minor": 1,
-        "Patch": 16
+        "Patch": 20
```

Comment:
You can undo this change. Once the latest version is published to the marketplace, I will push up the latest patch version number to `master`.

Reply:
done